packages/edit-site/src/components/global-styles/shadows-panel.js

```diff
@@ -135,9 +135,17 @@ function ShadowItem( { shadow, category } ) {
 	return (
 		<NavigationButtonAsItem
 			path={ `/shadows/edit/${ category }/${ shadow.slug }` }
-			icon={ shadowIcon }
 		>
-			{ shadow.name }
+			<HStack direction="row">
+				<FlexItem className="edit-site-shadows-panel__item">
```

Comment:
I think there's no need for this class. The editor doesn't target this element with custom styles. What was the reason for introducing it?

Now that the navigation button doesn't display the icon, we can simplify the markup.

```js
<HStack>
  <FlexItem>{ shadow.name }</FlexItem>
  <FlexItem display="flex">
	  <Icon icon={ isRTL() ? chevronLeft : chevronRight } />
  </FlexItem>
</HStack>
```

Reply:
Hi @Mamaduka, done with the changes. The PR is now ready to be reviewed again. Thanks.

Reply:
Thanks, @yogeshbhutkar! Let's wait for final feedback on the issue, and then I think we'll be good to merge.